src/Login.vue

```diff
@@ -0,0 +1,54 @@
+<script setup lang="ts">
+import { onMounted, reactive, ref, type Ref } from "vue";
+import { useSessionStore } from "./stores/sessionState";
+
+const sessionConfig = useSessionStore();
+onMounted(() => {
+  fetch('http://localhost:9000/users', {
```

Comment:
Might be worth having the references to the backend set in an environment variable. Vite does have support for them, with an example of them being used [here](https://github.com/UKCloud/CMPMP/blob/master/src/components/VaunchAbout.vue#L5 ). Vite environment variables require the prefix `VITE_APP_` however, so something like `VITE_APP_BACKEND_URL` would be appropriate for this